server.js

```diff
@@ -1,61 +0,0 @@
-'use strict';
```

Comment:
I looked into the original commits where the files were moved and renamed, and Git appears to know that they're moved. I think it might be a problem with the way GitHub does diffs.

Reply:
I checked with SourceTree as well. It looks like it recognised the file rename in the commit where it was renamed, but if use SourceTree to compare the latest commit to master, it forgets. No idea why.

Dumb git.